prime-router/metadata/hl7_mapping/datatypes/Address/XAD.yml

```diff
@@ -53,6 +53,15 @@ elements:
     hl7Spec: [ '%{hl7SpecXADField}-11' ]
 
   # TODO: #11817
+  # TODO: #11664
+  # XAD.12, XAD.13, XAD.14
+  - name: period
```

Comment:
This doesn't necessarily result in byte-for-byte equivalence right? Period could have been populated by fields 13 and 14 and, if that's the case, then here you are setting XAD 12 to the values of 13 and 14 when the original HL7v2 did not have XAD 12 set in the first place. Sounds like something that can be solved by extensions for XAD 12, but I'm curious if we think this is worth it. My opinion is the added complexity is not worth it and this is fine as is. @mkalish thoughts?